src/main/java/seedu/recipe/model/goal/GoalCount.java

```diff
@@ -39,4 +39,9 @@ public boolean equals(Object other) {
                 && goal.goalName.equals(((GoalCount) other).goal.goalName)
                 && count.equals(((GoalCount) other).count)); // state check
     }
+
+    @Override
+    public String toString() {
```

Comment:
These lines are for debugging the pie chart issue (to verify that undo/redo function + the tag counter is indeed working correctly. To remove during final product / once bug is fixed.